apps/teamsjs-cc-app/tabs/src/components/capabilities/pages/ShareDeepLink.tsx

```diff
@@ -0,0 +1,41 @@
+import * as Fluent from "@fluentui/react-northstar";
+
+import { app, pages } from "@microsoft/teams-js";
+
+/**
+ * This component returns a button which share's deep link.
+ */
+export const ShareDeepLink = () => {
+  // cCheck if app is initialized;
+  if (app.isInitialized()) {
+    return (
+      <>
+        {pages.isSupported() && (
+          <Fluent.Segment className="ui-pagessegment">
+            <Fluent.Header content="Share Deep Link" as="h2" />
+            <Fluent.Flex gap="gap.small" vAlign="center">
+              <Fluent.Text
+                className="ui-pagestext"
+                content="Shares a deep link that a user can use to navigate back to a specific state in this page. Please note that this method does yet work on mobile hosts."
```

Comment:
Is the fact that this doesn't work on mobile hosts yet something we're following up with them on?

Reply:
We have already and this is being worked on atm